test/unit/extract/xbrl_test.py

```diff
@@ -99,28 +101,21 @@ def test_xbrl2sqlite(settings, forms, mocker, tmp_path):
     mock_datastore = mocker.MagicMock()
     mocker.patch("pudl.extract.xbrl.FercXbrlDatastore", return_value=mock_datastore)
 
-    # always use tmp path here so that we don't clobber the live DB when --live-dbs is passed
-    mock_pudl_paths = mocker.MagicMock(
-        spec=PudlPaths(),
-        sqlite_db_path=lambda form_name: tmp_path / f"{form_name}.sqlite",
-        output_dir=PudlPaths().output_dir,
-    )
-    mocker.patch("pudl.extract.xbrl.PudlPaths", return_value=mock_pudl_paths)
-
-    # Construct xbrl2sqlite op context
-    context = build_op_context(
+    # always use tmp ath here so that we don't clobber the live DB when --live-dbs is passed
```

Comment:
We're no longer mocking `PudlPaths()` here - is this comment still relevant? Seems like we're checking at test runtime for `--live-dbs`, which is fine I guess but also error-prone - we need to remember to bail out of any db-touching test when `live_dbs == True`.

Reply:
Yeah, the fact that we have the tests operating in these two distinct modes (read-only vs mutating) is a bit dangerous IMO and there's always a possibility that we don't do isolation well and things get mangled. I suppose that, ideally, we would rely on most of the db tests to be validations that will not modify whatever is in place.